### PR TITLE
Bael 3464 ninja

### PR DESCRIPTION
**Pull request by SmartyAnsh**

### CODE REVIEW

File: ninja/src/main/java/controllers/ApplicationController.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    @Inject
+    Lang lang;
+
+    @Inject
+    Messages msg;
+    
+    private static Log logger = LogFactory.getLog(ApplicationController.class);
+    
+    @Inject 
+    Provider<EntityManager> entitiyManagerProvider;
```
Comment (KevinGilmore) [inline ninja/src/main/java/controllers/ApplicationController.java]: rename this to entityManagerProvider (there's an extra 'i' in the name)

File: ninja/src/test/java/controllers/ApiControllerDocTesterTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package controllers;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.Assert.assertThat;
+import org.doctester.testbrowser.Request;
+import org.doctester.testbrowser.Response;
+import org.junit.Test;
+import ninja.NinjaDocTester;
+
+public class ApiControllerDocTesterTest extends NinjaDocTester {
```
Comment (KevinGilmore) [inline ninja/src/test/java/controllers/ApiControllerDocTesterTest.java]: question: is there a reason not to follow the naming convention for test classes, i.e. the class names should end in "UnitTest" or "IntegrationTest", as appropriate

File: ninja/src/test/java/controllers/ApiControllerMockTest.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import javax.inject.Inject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import ninja.NinjaRunner;
+import ninja.Result;
+import services.UserService;
+
+@RunWith(NinjaRunner.class)
+public class ApiControllerMockTest {
```
Comment (KevinGilmore) [inline ninja/src/test/java/controllers/ApiControllerMockTest.java]: same question about the test class name

File: pom.xml [hunk 4]
```diff
@@ -589,7 +589,7 @@
                 <!-- <module>muleesb</module> --> <!-- Fixing in BAEL-10878 -->
                 <module>mustache</module>
                 <module>mybatis</module>
-
+				<module>ninja</module>
```
[the remark this answers is not in the document]
Comment (SmartyAnsh) [reply pom.xml]: fixed

File: pom.xml [hunk 5]
```diff
@@ -1223,7 +1224,7 @@
...
```
[the remark this answers is not in the document]
Comment (SmartyAnsh) [reply pom.xml]: fixed

File: ninja/src/main/java/controllers/ApplicationController.java (hunk 1 above)
Comment (SmartyAnsh) [reply ninja/src/main/java/controllers/ApplicationController.java]: typo fixed

File: ninja/src/test/java/controllers/ApiControllerDocTesterTest.java (hunk 2 above)
Comment (SmartyAnsh) [reply ninja/src/test/java/controllers/ApiControllerDocTesterTest.java]: Fixed to ApiControllerDocTesterUnitTest
Ninja only has unit test (even for integration test) and Test class is created by the framework be default.

File: ninja/src/test/java/controllers/ApiControllerMockTest.java (hunk 3 above)
Comment (SmartyAnsh) [reply ninja/src/test/java/controllers/ApiControllerMockTest.java]: Fixed to ApiControllerMockUnitTest
Ninja only has unit test (even for integration test)

**Pull request merged**